tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
+    "should successfully parse" >> {
 
-  "parseRelFile - ./image.png" in {
-    parseRelFile("./image.png") must beSome(file("image.png"))
-  }
+      """simple file name with extension such as "image.png"""" in {
```

Comment:
I kind of like it. It conveys the intent of the test. At the same time, when looking at the spec or when running it, it's easier to parse/understand if there is an example value. I don't think it's any more misleading than if the example wasn't there. Both sound like they should test more than a single value. I am not 100% sold either, but to me it seems like the best of the three.